Sources/Basics/Triple+Basics.swift

```diff
@@ -0,0 +1,198 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2014-2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import enum TSCBasic.JSON
```

Comment:
One last request, but otherwise this LGTM: can we have a comment in either of these files or in README.md of this directory that these files are vendored from Swift Driver, maybe also with an explanation why? Without it I don't think we make it obvious why the directory is called `Vendor` in the first place without opening this PR and reading this discussion to get the context.

Reply:
Yeah for sure, I'll push that diff later tonight/tomorrow

Reply:
done!